### DISCUSSION TITLE
Sign SDK v2 final now available 🎂🚀

### DISCUSSION THREAD

**Discussion by chadyj** [WalletConnect/walletconnect-monorepo#1266] [Announcements]
The WalletConnect Sign SDK v2 release candidate just shipped.

Read our [launch blog post](https://medium.com/walletconnect/calling-all-devs-the-walletconnect-v2-0-sign-beta-is-over-d7bb786cdd93).

Here are some quick links

- JS https://github.com/WalletConnect/walletconnect-monorepo/releases
- Kotlin https://github.com/WalletConnect/WalletConnectKotlinV2/releases
- Swift https://github.com/WalletConnect/WalletConnectSwiftV2/releases
- Docs https://docs.walletconnect.com/2.0/introduction/sign

**Comment by silvestrst-crypto** [comment]
@chadyj Are there any plans/work-in-progress for WalletConnect v2 Rust client?

**Reply by chadyj** [reply]
No immediate plans.  Can you tell us more about the use case you have in mind?

**Reply by silvestrst-crypto** [reply]
I am working on a pure Rust project, where we intend to use WalletConnect v2 for DApp communication, and thus pure Rust client would be great.

**Reply by SnailMyth** [reply]
if this v2 have its testing example website?like v1.

**Reply by silvestrst-crypto** [reply]
> if this v2 have its testing example website?like v1.

Yup, there are decent amount of example wallets and DApps.

This for example: https://github.com/WalletConnect/web-examples
